### DISCUSSION TITLE
Support for Google Takeout in TGZ format

### DISCUSSION THREAD

**Discussion by windmueller** [ente-io/ente#648] [Enhancements]
Currently the desktop app only supports the ZIP format for Google Takeout. Since the takeout feature limits ZIP archives to 2GB per file, this can be cumbersome for large exports.

The TGZ format for takeout has a limit of 50GB per file. I was able to convert the file locally from ZIP to TGZ, but that should not be necessary.

**Comment by s38b35M5** [comment]
This would be great to have supported, as Google gives a gentle warning that nudges the user to `.tgz` for files above 2GB.

Just be advised @windmueller that Google Takeout supports ZIP file creation up to 50GB, with the disclaimer that the ZIP files above 2GB are created in ZIP64 format, which could result in edge cases of incompatibility, but are handled without problem by ente desktop.

![image](https://github.com/ente-io/photos-desktop/assets/138058715/311e00f7-3f7b-4f3b-acf7-a1fdde9029f9)

**Comment by windmueller** [comment]
@s38b35M5 Thanks for the clarification, you are right of course. So support for TGZ would be nice but a low priority is totally okay for that feature.

**Comment by tgf9** [comment]
I'm poking around Ente trying to see if it's something I could use. I just did a large export from Google Takeout, which took a long time. And now I'm bummed that I can't import the TGZ format. Not a great initial user experience. Just wanted to add another data point.

**Reply by vishnukvmd** [reply]
Instead of dealing with Takeout again, if your disk allows for it, please unzip all of the archives into a single folder and import that.

**Reply by tgf9** [reply]
OK, thanks for the tip. I can try that. :+1: 
Also, for your consideration: https://github.com/ente-io/ente/pull/1067
I thought it would be helpful to tell the user what's actually supported.

**Comment by s38b35M5** [comment]
> Instead of dealing with Takeout again, if your disk allows for it, please unzip all of the archives into a single folder and import that.

It should be noted that my initial import (83GB in 10GB or 20GB archives) was problematic, and the support team recommended that my experience was related to not simply letting Ente ingest the takeout archive as one file.
